packages/components-js/scripts/convertToNextJsVRTPage.ts

```diff
@@ -1,20 +1,32 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import { paramCase } from 'change-case';
 import { convertToReactVRTPage, ReactCharacteristics } from './convertToReactVRTPage';
+import { paramCase } from 'change-case';
 
 const sourceBasePath = path.resolve(__dirname, '../../components-react/src');
 const pollComponentsReadyFilePath = path.resolve(sourceBasePath, 'pollComponentsReady.ts');
-const pollComponentsReadyFileContent = fs.readFileSync(pollComponentsReadyFilePath, 'utf8');
+const pollComponentsReadyFileContent = fs
+  .readFileSync(pollComponentsReadyFilePath, 'utf8')
+  .replace(/export (const pollComponentsReady)/, '$1');
 
 export const convertToNextJsVRTPage = (
   ...params: [string, string, string, string, string, string, ReactCharacteristics]
 ): { fileName: string; fileContent: string } => {
   const { fileName: convertedFileName, fileContent: convertedFileContent } = convertToReactVRTPage(...params);
 
+  const pagesWithClientHooks: string[] = [];
+  const fileName = paramCase(convertedFileName.replace(/\.tsx/, ''));
+  convertedFileContent.match(/use[A-Z]/) && pagesWithClientHooks.push(fileName);
```

Comment:
Why do we need this array after all? Why not just assign this to a meaningful constant and use it afterwards?